### PR TITLE
Update add-item routes

### PR DESCRIPTION
**Pull request by Nirmalya24**
- Fix nasty query bug in `checkItemInSection()` in `MenuModel.ts`
- Update POST `/api/items/add-item`

### CODE REVIEW

File: App.ts [hunk 1]
```diff
@@ -167,50 +216,64 @@ class App {
...
+     */
     router.post('/api/items/add-item', (req, res) => {
       // Get the RestaurantId, menuId, sectionName, itemId from req body
-      let restaurantId: string = req.body.restaurantId;
-      let menuId: string = req.body.menuId;
-      let sectionName: string = req.body.sectionName;
-      let itemId: string = req.body.itemId;
+      let restaurantID: string = req.body.restaurantID;
+      let menuID: string = req.body.menuID;
+      let sectionName: string = req.body.menuSection;
```
[the remark this answers is not in the document]
Comment (Nirmalya24) [reply App.ts]: Done

File: model/ItemModel.ts [hunk 2]
```diff
@@ -57,20 +57,20 @@ class ItemModel {
...
+     * check a single item by ID
      *@param filter - filter object
-     *@returns - JSON Object
+     *@returns - true if item exist.
      */
-   public retrieveItemByID(response: any, filter: Object) {
-       // TODO: Check if this is working
-       console.log("Query single item with filter: " + JSON.stringify(filter));
+   public checkItem(filter: Object):any {
+       console.log("check if item"+filter+" in the Items collection");
```
Comment (goyaladit) [inline model/ItemModel.ts:65]:
```suggestion
       console.log("check if item "+filter+" in the Items collection");
```

Comment (Nirmalya24) [reply model/ItemModel.ts]: Updated console.logs()

**Review by goyaladit — DISMISSED** [review]
A couple very minor changes

**Review by troy8chen — APPROVED** [review]
Looks good to me.

**Review by jen-chieh — APPROVED** [review]
good to me

**Pull request merged**